src/Bridge/Symfony/Resources/config/doctrine_mongodb_odm.xml

```diff
@@ -71,7 +72,7 @@
         <service id="fidry_alice_data_fixtures.persistence.persister.doctrine_mongodb.object_manager_persister"
                  class="Fidry\AliceDataFixtures\Bridge\Doctrine\Persister\ObjectManagerPersister"
                  lazy="true">
-            <argument type="service" id="doctrine_mongodb.odm.document_manager" />
+            <argument>%doctrine_mongodb.odm.document_managers%</argument>
```

Comment:
that won't work. As you made the constructor variadic, you need to add many arguments.

and btw, this parameters contains the *names* of document managers, not the objects (a parameter cannot contain references to services)